packages/feedback/src/core/integration.ts

```diff
@@ -176,25 +176,33 @@ export const buildFeedbackIntegration = ({
     };
 
     const _loadAndRenderDialog = async (options: FeedbackInternalOptions): Promise<FeedbackDialog> => {
+      const screenshotRequired = options.showScreenshot && isScreenshotSupported();
       const [modalIntegration, screenshotIntegration] = await Promise.all([
         _findIntegration<FeedbackModalIntegration>('FeedbackModal', getModalIntegration, 'feedbackModalIntegration'),
-        showScreenshot && isScreenshotSupported()
+        screenshotRequired
           ? _findIntegration<FeedbackScreenshotIntegration>(
               'FeedbackScreenshot',
               getScreenshotIntegration,
               'feedbackScreenshotIntegration',
             )
           : undefined,
       ]);
-      if (!modalIntegration || (showScreenshot && !screenshotIntegration)) {
```

Comment:
this line was preventing the modal from rendering on a real mobile device if `showScreenshot:true` was set.

why?
we asked for screenshots,
but it's not supported, so we didn't load it
and this like says "we expected it to be loaded, but it's not"

fixed now!